include/TrackContainerView.h

```diff
@@ -167,6 +168,7 @@ public slots:
 
 protected:
 	static const int DEFAULT_PIXELS_PER_BAR = 16;
+	static const int DEFAULT_TRACK_HEIGHT = 32;
```

Comment:
```suggestion
	static constexpr int DEFAULT_TRACK_HEIGHT = 32;
```

Reply:
I would remove this constant entirely - there are already `MINIMAL_TRACK_HEIGHT` and `DEFAULT_TRACK_HEIGHT` constants defined in `include/Track.h`.